app/helpers/ical_helper.rb

```diff
@@ -10,7 +10,7 @@ def icalendar(*events)
           item.dtstart     = event.date
           item.dtend       = event.end_date
           item.url         = event_url(event)
-          item.location    = event.location.name if event.location
+          item.location    = event.location.calendar_event_address if event.location
```

Comment:
To ensure that this feature will work in the future when the code base changes. Could you please add a test for this?

Reply:
Do we need explicit tests for this or would the smoke testing in the events controller specs surfice, given that we already have the calendar event address test?